src/app/home-body/home-body.component.html

```diff
@@ -52,10 +52,7 @@
         <div>
           <md-card>
             <md-card-content>
-              <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
-                magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-                consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-                Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+              <p> Click here to start he learning process!!!
```

Comment:
These boxes need to be filled in with a lot more. This is also outside the scope of the toolbar work.